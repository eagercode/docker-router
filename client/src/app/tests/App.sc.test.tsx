import * as React from 'react';
import { mount } from 'enzyme';
import 'jest-styled-components';
import Div from '../App.sc';

describe('<Div/>', () => {
    it('app styled div snapshot', () => {
        expect(mount(<Div/>)).toMatchSnapshot();
    });
});
