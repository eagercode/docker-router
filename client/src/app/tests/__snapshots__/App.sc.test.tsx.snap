// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Div/> app styled div snapshot 1`] = `
.c0 * {
  font-family: Roboto,sans-serif;
}

<styled.div>
  <div
    className="c0"
  />
</styled.div>
`;
